### ISSUE TITLE
Framebuffer autosize() method may be broken

### ISSUE THREAD

**Issue by nickmcintyre** [processing/p5.js#6998]
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [ ] Core/Environment/Rendering
- [ ] Data
- [ ] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [X] WebGL
- [ ] Build process
- [ ] Unit testing
- [ ] Internationalization
- [ ] Friendly errors
- [ ] Other (specify if possible)

### p5.js version

1.9.3

### Web browser and version

Chrome 124.0.6367.91

### Operating system

macOS 14.4.1

### Steps:
1. Call `myBuffer.autoSized()`
2. Get `TypeError: myBuffer.autoSized is not a function`.

Here's the [full sketch](https://editor.p5js.org/mcintyre/sketches/yDqameyNv).

My first guess is that there's a name collision between the `autoSized()` method and the property which is a `Boolean`. Here's the [implementation](https://github.com/processing/p5.js/blob/7f3b48a78647e6c39da29163b7353b668acfef3a/src/webgl/p5.Framebuffer.js#L272) for reference:

```js
autoSized(autoSized) {
  if (autoSized === undefined) {
    return this.autoSized;
  } else {
    this.autoSized = autoSized;
    this._handleResize();
  }
}
```

**Comment by davepagurek** [comment]
Good catch @nickmcintyre! I think that is indeed the cause. I'll rename the internal variable to remove the clash.
